src/main/java/com/epam/machine/util/LoginCheck.java

```diff
@@ -1,12 +1,13 @@
 package com.epam.machine.util;
 
 public final class LoginCheck {
-    public static boolean checkLog (String username, String password) {
```

Comment:
If u implemented method allows get client by login, may be its time to remove this class and check password into servlet?